src/main/scala/org/broadinstitute/clio/model/authorization.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.clio.model
+
+import akka.http.scaladsl.model.headers.{ModeledCustomHeader, ModeledCustomHeaderCompanion}
+
+import scala.util.Try
+
+
+trait OidcHeader {
```

Comment:
I'm not sure what our conventions are in Scala, but I didn't expect to see a class named `OidcHeader` in a file called `authorization.scala`. I don't know why `Oidc` is; is that something specific to this implementation or a general concept about authorization?

There should be comments here too.